MapboxNavigation/DayStyle.swift

```diff
@@ -233,18 +233,31 @@ open class NightStyle: DayStyle {
     }
 }
 
+#if canImport(CarPlay)
+@available(iOS 12.0, *)
+public protocol CarPlayStyle {
+    var previewStyleURL: URL { get }
+}
 
-open class CarPlayDayStyle: DayStyle {
+@available(iOS 12.0, *)
+open class CarPlayDayStyle: DayStyle, CarPlayStyle {
+    public var previewStyleURL = URL(string: "mapbox://styles/mapbox/navigation-preview-day-v4")!
+    
     open override func apply() {
         super.apply()
         ExitView.appearance().foregroundColor = .white
         GenericRouteShield.appearance().foregroundColor = .white
```

Comment:
@1ec5 `CarPlayDayStyle` should apply: 
```
ExitView.appearance().foregroundColor = .black
GenericRouteShield.appearance().foregroundColor = .black
```

Reply:
See reference: 
[Black foreground color for Day Styles' ExitView](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/DayStyle.swift#L91)
[GenericRouteShield.appearance().foregroundColor = .black](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/DayStyle.swift#L97)

Reply:
The background is dark on CarPlay, so we need to override the DayStyle to get some contrast.